src/types/api/rpcspec_0_6/contract.ts

```diff
@@ -11,7 +11,7 @@ export type ABI = Array<
   FUNCTION | CONSTRUCTOR | L1_HANDLER | EVENT | STRUCT | ENUM | INTERFACE | IMPL
 >;
 
-export type FUNCTION = {
+type FUNCTION = {
```

Comment:
I believe removing exports for types makes the affected types inaccessible for the API documentation generation.

I doubt anyone actually uses/reads it down to this level and the existing docs probably have more dead-end references, but I think having the reference available is better than not having it.

Reply:
Hmm, these types exports were never accessible because in the `index` file they were never re-exported (`Methods` and `ABI` were exported only 👉  [link](https://github.com/starknet-io/starknet.js/pull/1193/files#diff-149d2fb59cd50a587db52e2e50bc00bf56fa5d3eed47c05b754aa3fc636e91edL5)).
And ofc these exports were never used within the pkg itself
cc @penovicp

Reply:
Good point, these RPC 0.6 types are already affected with the issue I mentioned because of the missing re-export. Just to show what I meant, this is the [ABI](https://starknetjs.com/docs/API/namespaces/types.RPC.RPCSPEC06#abi) doc that references them, and this is the [ABI](https://starknetjs.com/docs/API/namespaces/types.RPC.RPCSPEC07.API#abi) doc for 0.7 where the references exist.

Let's leave them unexported, 0.6 shouldn't be around much longer anyway.